scss/mixins/_responsive.scss

```
@@ -0,0 +1,7 @@
+@import '../variables/size';
+
+@mixin responsive($breakpoint) {
```

Phân biệt rõ ràng ra giữa responsive max-width và min-width, dùng cả 2 loại này sẽ thuận tiện hơn trong nhiều trường hợp khác nhau
```suggestion
@mixin maxWidth($breakpoint) {
@mixin minWidth($breakpoint) {
```